package.json

```diff
@@ -23,6 +23,7 @@
     "node-fetch": "^2.1.2",
     "nugget": "^2.0.1",
     "octicons": "^7.3.0",
+    "pluralize": "^7.0.0",
```

Comment:
Adding an entire dependency to add an `s` is a bit rough

Reply:
Removed